### ISSUE TITLE
Not working on Debian 12

### ISSUE THREAD

**Issue by SrArminho** [laurento/gnome-shell-extension-ideapad#32]
It's been a few days since I installed Debian 12 on my Lenovo IdeaPad 3i.
The extension never worked on this Debian. I already checked `/etc/sudoers.d/ideapad`
I also checked `/sys/bus/platform/drivers/ideapad_acpi/VPC2004:00/conservation_mode`
Everything seems to be correct

I tried `echo "ideapad_laptop" | sudo tee -a /etc/modules`
I tried another Kernel version...
But nothing works

I tried and works on Fedora 37, Arch Linux and Ubuntu 22.04 and 22.10

I can turn Conservation Mode on or off with `sudo bash -c 'echo 0 > /sys/bus/platform/drivers/ideapad_acpi/VPC2004:00/conservation_mode` or `sudo bash -c 'echo 1 > /sys/bus/platform/drivers/ideapad_acpi/VPC2004:00/conservation_mode`

---
### These are some errors I encountered when trying to use the extension:
![image](https://user-images.githubusercontent.com/121321534/232245356-587f3640-0865-476a-95f2-f458f60713b9.png)

**Comment by laurento** [comment]
Hi there! I use this extension on Debian without any issues ;) According to the logs you shared, your sudo configuration seems to be the problem.

When everything is set up as expected, you should be able to run the following command from the terminal without being prompted for password.

`/bin/sh -c 'echo 0 | sudo tee /sys/bus/platform/drivers/ideapad_acpi/VPC2004:00/conservation_mode'`

**Comment by SrArminho** [comment]
Yes, there was a typo in the sudoers file. You can close this issue
